src/interface/src/app/home/about/about.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-planscape-about',
+  templateUrl: './about.component.html',
+  styleUrls: ['./about.component.scss'],
+})
+export class AboutComponent {
+  readonly text1: string = `
```

Comment:
Why not have this in the template verbatim?

Reply:
Done.